homeassistant/components/verisure/coordinator.py

```diff
@@ -43,10 +44,12 @@ def __init__(self, hass: HomeAssistant, entry: ConfigEntry) -> None:
     async def async_login(self) -> bool:
         """Login to Verisure."""
         try:
-            await self.hass.async_add_executor_job(self.verisure.login)
-        except VerisureError as ex:
+            await self.hass.async_add_executor_job(self.verisure.login_cookie)
+        except VerisureLoginError as ex:
+            LOGGER.error("Could not log in to verisure, %s", ex)
+            raise ConfigEntryAuthFailed("Credentials expired for Verisure") from ex
+        except VerisureResponseError as ex:
```

Comment:
The cal in `async_setup_entry` should raise an `ConfigEntryNotReady` in case this happens